tck/features/expressions/path/Path3.feature

```diff
@@ -0,0 +1,82 @@
+#
+# Copyright (c) 2015-2020 "Neo Technology,"
+# Network Engine for Objects in Lund AB [http://neotechnology.com]
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Attribution Notice under the terms of the Apache License 2.0
+#
+# This work was created by the collective efforts of the openCypher community.
+# Without limiting the terms of Section 6, any Derivative Work that is not
+# approved by the public consensus process of the openCypher Implementers Group
+# should not be described as “Cypher” (and Cypher® is a registered trademark of
+# Neo4j Inc.) or as "openCypher". Extensions by implementers or prototypes or
+# proposals for change that have been documented or implemented should only be
+# described as "implementation extensions to Cypher" or as "proposed changes to
+# Cypher that are not yet approved by the openCypher community".
+#
+
+#encoding: utf-8
+
+Feature: Path3 - Length of a path
+
+  Scenario: Pass the path length test
```

Comment:
I am torn on this one and the next one. I see where you coming from, but I like to avoid duplication as much as possible. The problem is, we have not yet a clear idea how to categorize `WHERE`. Right now I am thinking, we could categorize `WHERE` into stuff like filtering on atomic values, compound values, node characteristics, relationship characteristics, path characteristics, aggregates, complex predicates, etc. 

In that case, these scenarios should go into `WHERE` filtering on path characteristics.

Reply:
removed
only on match6-2